files/modrinth-exclude-include.json

```diff
@@ -45,6 +45,7 @@
     "iris-flywheel",
     "ItemBorders",
     "ItemLocks",
+    "just-zoom",
```

Comment:
Mentioned in discord also, but filename doesn't have dash

![image](https://github.com/user-attachments/assets/f79f2db3-5f78-465b-af26-9047d5db9e6e)